.github/dependabot.yml

```diff
@@ -1,12 +0,0 @@
-version: 2
```

Comment:
Why delete dependabot?  I agree that the current config is a little noisy.  Here's the config I'm using to only get notified of semver major updates:

```
version: 2
updates:
  - package-ecosystem: "npm"
    directory: "/"
    schedule:
      interval: "weekly"
    ignore:
      - dependency-name: "*"
        update-types:
          ["version-update:semver-patch", "version-update:semver-minor"]
```

Can we keep it?

Reply:
Added it back, it just got overlooked when doing this rewrite (from a fresh start).